substrate/frame/revive/src/evm/tracing/prestate_tracing.rs

```diff
@@ -172,7 +189,11 @@ where
 	}
 
 	fn instantiate_code(&mut self, code: &crate::Code, _salt: Option<&[u8; 32]>) {
-		self.is_create = Some(code.clone());
+		if let Some(current) = self.calls.last_mut() {
+			current.create_code = Some(code.clone());
+		} else {
+			self.create_code = Some(code.clone());
```

Comment:
Don't we actually need just the else branch ? Since we seem to always call instantiate_code before calling enter_child_span.

Reply:
correct